MonoGame.Framework/Game.cs

```diff
@@ -632,11 +632,16 @@ private void Platform_ApplicationViewChanged(object sender, ViewStateChangedEven
         internal void applyChanges(GraphicsDeviceManager manager)
         {
 			Platform.BeginScreenDeviceChange(GraphicsDevice.PresentationParameters.IsFullScreen);
+
+#if !(WINDOWS && DIRECTX)
+
+            // FIXME : FULLSCREEN

```

Comment:
What is this FIXME for?  What code is missing here?  I'm ok with adding notes like this, but they need to clearly explain what is needed to be done.